### PR TITLE
Star glyph not rendering correctly

### LINKED ISSUE #4515: Star glyph not rendering correctly
- Gitea version (or commit ref): 32f2559
- Operating system: (whatever try.gitea.io runs on)
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes [https://try.gitea.io/Arrow/TestRepo/issues/1](https://try.gitea.io/Arrow/TestRepo/issues/1)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description
The glyph icon for "Star" renders incorrectly on all browsers I tried. I am assuming it has something to do with unicode, as I have seen this issue before related to that. It is possible this affects other glyphs around the gui, but can't say for sure.

## Screenshots
image

### PR DESCRIPTION
**Pull request by Wqrld**
fix #4515

### CODE REVIEW

**Comment by Wqrld** [comment]
Fixed CI

**Comment by Wqrld** [comment]
CI failed on mysql but that seems to be just a cloudfront outage.

EDIT: packagelock seems to have changed, is that a problem?

File: package-lock.json [hunk 1]
```diff
@@ -9,10 +9,10 @@
             "dev": true,
```
Comment (techknowlogick) [inline package-lock.json:9]: package lock updates aren't needed to change the CSS

**Comment by techknowlogick** [comment]
CI restarted

**Comment by lafriks** [comment]
I don't think that is correct way to fix this. I'm on mobile can't check right now but `fa` class should set correct font family already

File: package-lock.json (hunk 1 above)
Comment (Wqrld) [reply package-lock.json:9]: yes, this hapenned when npm installing less

Comment (techknowlogick) [reply package-lock.json:9]: with git you can:
```
git add public/css/index.css
git add  public/less/_repository.less
git commit
```
that will only add specific files to your commit.

**Comment by exside** [comment]
@lafriks agreed, but there is some strange interference between the Semantic UI iconfont and fontawesome...guess that's causing all the trouble.

**Comment by atom0s** [comment]
Simply changing the icon type from fa-star-o to fa-star fixes this rather than a bunch of file edits.

**Comment by techknowlogick** [comment]
@atom0s sadly that change also isn't entirely effective, as it would break logic: `<i class="icon fa-star{{if not $.IsStaringRepo}}-o{{end}}"></i>` is the existing logic, and changing it would mean that a user couldn't tell if they are have starred the repo yet.

**Comment by lafriks** [comment]
Does not changing to `<i class="icon fa fa-star{{if not $.IsStaringRepo}}-o{{end}}"></i>` help?

**Comment by techknowlogick** [comment]
@lafriks that specific change didn't work, however I found replacing `icon` with `fa` worked. Even though it did work I'm still unsure about it because `icon` is used everywhere else in the application.

**Comment by lafriks** [comment]
@techknowlogick @Wqrld it is caused by semantic ui update.
This is correct solution for unstarred use: `<i class="icon star outline"></i>` for starred: `<i class="icon star"></i>`

So template file should have:
```
<i class="icon star{{if not $.IsStaringRepo}} outline{{end}}"></i>
```

**Comment by techknowlogick** [comment]
That's great news! I should be waking up in a few hours and can submit a PR with that change, or if you get to it before me I can give it my LG-TM

**Comment by lafriks** [comment]
I have submitted fix, closing this

**Pull request closed without merging**